### PR TITLE
feat: nutanix licence tile

### PR DESCRIPTION
**Pull request by radireddy**
| Question         | Answer
| ---------------- | ---
| Branch?          | feat/nutanix
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-7851
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/modules/nutanix/src/dashboard/component/licence-tile/licences.class.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  getFeatures() {
+    return this.features;
+  }
+
+  getFeatureByName(featureName) {
+    return this.features.find((feature) => feature.getName() === featureName);
+  }
+
+  isFeatureAvailable(featureName) {
+    return this.getFeatureByName(featureName) !== null;
```
Comment (JeremyDec) [inline packages/manager/modules/nutanix/src/dashboard/component/licence-tile/licences.class.js:19]:
```suggestion
    return this.getFeatureByName(featureName) !== undefined;
```

or 
```suggestion
    return this.getFeatureByName(featureName) != null;
```

File: packages/manager/modules/nutanix/src/dashboard/component/licence-tile/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "nutanix_cluster_licence_title": "Licences Nutanix",
+  "nutanix_cluster_licence_enabled": "Activar",
+  "nutanix_cluster_licence_disabled": "Desactivar"
```
Comment (JeremyDec) [inline packages/manager/modules/nutanix/src/dashboard/component/licence-tile/translations/Messages_fr_FR.json:4]:
```suggestion
  "nutanix_cluster_licence_enabled": "Activer",
  "nutanix_cluster_licence_disabled": "Désactiver"
```

File: packages/manager/modules/nutanix/src/dashboard/general-info/routing.js [hunk 3]
```diff
@@ -31,11 +31,13 @@ export default /* @ngInject */ ($stateProvider) => {
         });
         return promise;
       },
-      handleError: /* @ngInject */ (Alerter) => (error) =>
+      handleError: /* @ngInject */ (Alerter) => (error) => {
+        console.log(error);
```
Comment (JeremyDec) [inline packages/manager/modules/nutanix/src/dashboard/general-info/routing.js:35]:
```suggestion
```

**Review by JeremyDec — CHANGES_REQUESTED** [review]

File: packages/manager/modules/nutanix/src/dashboard/component/licence-tile/controller.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  /* @ngInject */
+  constructor(NutanixService) {
+    this.NutanixService = NutanixService;
+  }
+
+  $onInit() {
+    this.loadingTechnicalDetails = true;
+    this.loadTechnicalDetails(this.serviceId)
+      .then((res) => {
+        const licenceFeatures = isArray(res.license.features)
```
Comment (varun257) [inline packages/manager/modules/nutanix/src/dashboard/component/licence-tile/controller.js:15]:
```suggestion
.then(({ license }) => {
        const licenceFeatures = isArray(license.features)
```

**Review by varun257 — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Review by antleblanc — CHANGES_REQUESTED** [review]
Some checks were not successful

> CDS/UXMAN-manager-build-v2 — build-v2: Fail
> CDS/UXMAN-manager-test-v3 — test-v3: Fail

**Comment by radireddy** [comment]
> Some checks were not successful
> 
> > CDS/UXMAN-manager-build-v2 — build-v2: Fail
> > CDS/UXMAN-manager-test-v3 — test-v3: Fail

build is ok now, https://cds.ovhcloud.tools/project/UXMAN/workflow/manager/run/28472

**Pull request merged**